decisions/importer/helsinki.py

```diff
@@ -164,12 +167,20 @@ def _import_organization(self, info):
                     role=person_info['role'],
                 ))
 
-        if org_type in ['office_holder', 'trustee']:
+        if org_type in [Org.OFFICE_HOLDER, Org.TRUSTEE]:
             self.save_post(org)
         else:
             self.save_organization(org)
 
     def import_organizations(self, filename):
+        self.logger.info('Clearing organization class definitions')
+        OrganizationClass.objects.all().delete()
```

Comment:
If nothing is changed upstream, organization import should be an idempotent operation. This would result in all the organizations being deleted, as well. I'm afraid you need to perform a proper sync to figure out what is added, modified or deleted. Come to think of it, deletion should not happen, so only additions and modifications need to be detected.